docs/template_plugin/tests/functional/shared_tests_instances/behavior/preprocessing.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multi-device/multi_device_config.hpp"
+
+#include "behavior/preprocessing.hpp"
+
+using namespace BehaviorTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> inputPrecisions = {
+    InferenceEngine::Precision::U8,
+    InferenceEngine::Precision::FP32
+    //,InferenceEngine::Precision::FP16
```

Comment:
Why this case is commented?

Reply:
Because template plugin does not support it. 
Will remove.